### ISSUE TITLE
Bingo columns not rewarding Copper Hourglass

### ISSUE THREAD

**Issue by MissElyri** [ObliqueNetwork/Terra#2236]
TL;DR: Atlas not receiving hourglasses upon bingo column completion

**Describe the bug**
Players are not receiving copper hourglasses upon completion of Columns on the bingo card. Has been tested on all 3 servers with variants in results. Some players claiming they have not received them since the full server reset.

**To Reproduce**
Complete Bingo column
Be rewarded money
No hourglass

**Expected behavior**
Players to receive Copper Hourglasses when completing a Bingo column

Tested Servers: Atlas, Orion, Titan
![image](https://github.com/NebulaMC-GG/Bugs-and-Issues/assets/78369601/b1c7059f-619e-410e-8b0d-9f78c43599ff)
![image](https://github.com/NebulaMC-GG/Bugs-and-Issues/assets/78369601/72ffefc7-b677-4d97-848f-3bc6e118d118)
![image](https://github.com/NebulaMC-GG/Bugs-and-Issues/assets/78369601/e85e73a3-f0e2-489a-8fc2-28c47c1dc0ee)

Receives Hourglasses: ✅ ❌ 
Receives message of reward: ☑️  ✖️ 
Altas ❌  ✖️  
Orion ✅  ✖️ 
Titan: ✅  ✖️ 

Sunlit_gman1 (Atlas) screenshots:
![image](https://github.com/NebulaMC-GG/Bugs-and-Issues/assets/78369601/7d87ee01-2718-493c-a872-d1c9a5314a48)
 A row also would've completed a column with no feedback on reward given aside from 2k coins
![image](https://github.com/NebulaMC-GG/Bugs-and-Issues/assets/78369601/11fbd97b-35b7-44a6-a74b-68e5f6634027)

**Comment by Rasgnarok** [comment]
Completed, only Atlas was lacking proper safeguard against Pixelmon items.
